pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -61,8 +61,31 @@ const (
 
 	operationPollInterval        = 3 * time.Second
 	operationPollTimeoutDuration = 30 * time.Minute
+
+	defaultLBSourceRange = "0.0.0.0/0"
+
+	//Expected annotations for GCE
+	gceLBAllowSourceRange = "kubernetes.io/gce.loadbalancer.allow.source.range"
 )
 
+//Validation Functions for expecting annotations in GCE
+var gceAnnotationValidation = map[string]cloudprovider.ValidationFunc{
```

Comment:
why make this a module scoped global instead of simple a function? there isn't that much value in sharing this with cloudprovider/cloud IMO (or if you think there is, please explain in a comment above the ValidationFunc type)